atom/browser/api/atom_api_window.cc

```diff
@@ -584,8 +584,17 @@ void Window::SetFocusable(bool focusable) {
   return window_->SetFocusable(focusable);
 }
 
-void Window::SetProgressBar(double progress) {
-  window_->SetProgressBar(progress);
+void Window::SetProgressBar(double progress, mate::Arguments* args) {
+  std::string mode;
+
+  if (args && args->Length() == 2) {
```

Comment:
Every rule has a reason behind it, I don't agree all of them, but keeping a consistent style with Chromium would be good for people knowing Chromium to get familiar with Electron, and vice verso.

If you want to have a change with the code style, starting a discussion at the chromium-dev mailing list would be the best way, there are discussions on C++ style there from time to time, and the style guide changes when there is good reason.